Server/ConcordiaCurriculumManager/Repositories/DossierReviewRepository.cs

```diff
@@ -50,4 +51,20 @@ public async Task<bool> SaveApprovalStages(IList<ApprovalStage> stages)
             .ThenInclude(deletionRequests => deletionRequests.Course)
             .FirstOrDefaultAsync();
     }
+
+    public async Task<Dossier?> GetDossierWithApprovalStagesAndRequestsAndDiscussion(Guid dossierId)
+    {
```

Comment:
On the frontend, we'll be able to see the ID of the person who sent a message, but how can we retrieve their name or email to show it? I think the easiest option is to remove this method altogether and use the already existing `GetDossierByDossierId` in DossierRepository, but add the extra lines about including the discussion and messages. This way we'll include the group members and the frontend can extract their names from there.